frontend/src/lib/Quiver.test.ts

```diff
@@ -162,11 +162,13 @@ describe("Quiver", () => {
       })
 
       it("throws an exception if row index is out of range", () => {
-        expect(() => q.getCell(5, 0)).toThrow("Row index is out of range.")
+        expect(() => q.getCell(5, 0)).toThrow("Row index is out of range: 5")
       })
 
       it("throws an exception if column index is out of range", () => {
-        expect(() => q.getCell(0, 5)).toThrow("Column index is out of range.")
+        expect(() => q.getCell(0, 5)).toThrow(
+          "Column index is out of range: 5"
+        )
```

Comment:
Hm, why were the quiver tests affected by this change? I don't see an obvious connection between the two

Reply:
Ah, yes :) That's a lazy fix to get the CI pipeline running introduced by a commit to the feature branch. No other connection to the PR.